### DISCUSSION TITLE
Licensing Info?

### DISCUSSION THREAD

**Discussion by jedieaston** [microsoft/openjdk#2] [Q&A] [answered]
Hi there! 

I was creating a WinGet manifest and was wondering if there was any information about how these binaries are licensed? I scoured the landing page and there wasn't any information.

**Comment by brunoborges** [comment, marked as answer]
It is GPL2 with Classpath Exception.

**Comment by karianna** [comment]
https://openjdk.java.net/legal/gplv2+ce.html for the full detail on that
license.

Cheers,
Martijn

**Comment by brunoborges** [comment]
Regarding `winget`, it is coming soon! Real soon. See this PR: https://github.com/microsoft/winget-pkgs/pull/10174

**Comment by jedieaston** [comment]
Hooray! 

![image](https://user-images.githubusercontent.com/21368066/113781972-7b09d180-96ff-11eb-8251-311aaafacadd.png)


Thanks for your help.
